src/expandable-section/__tests__/interactions.test.tsx

```diff
@@ -23,15 +23,15 @@ describe('Expandable Section - Interactions', () => {
       wrapper = renderExpandableSection({ onChange: onChangeSpy });
     });
     test('toggles from false to true when header is clicked, change event fires', () => {
-      wrapper.findHeader().click();
```

Comment:
This would still work today because when no interactive elements are present the clickable area is not restricted, but it might change in the future, therefore the code in the right is the new recommended way.